src/store.rs

```diff
@@ -15,6 +15,7 @@ pub trait Storage: Debug {
     fn add_contract(&mut self, hash: U256, code: Vec<U256>) -> Result<(), StorageError>;
     fn storage_read(&self, key: StorageKey) -> Result<Option<U256>, StorageError>;
     fn storage_write(&mut self, key: StorageKey, value: U256) -> Result<(), StorageError>;
+    fn get_state_storage(&self) -> &HashMap<StorageKey, U256>;
```

Comment:
I believe the idea of the abstraction as it is made is that we ask for the value we need by calling `storage_read(key)` instead of returning the `HashMap`.